Packs/ILert/Integrations/ILert/ILert.py

```diff
@@ -0,0 +1,148 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+''' GLOBAL VARS '''
+# iLert API works only with secured communication.
+USE_SSL = not demisto.params().get('insecure', False)
+
+USE_PROXY = demisto.params().get('proxy', True)
```

Comment:
you don't have the insecure and proxy parameters in the configuration (in the yml) - please add them.
see [this example](https://github.com/demisto/content/blob/master/Packs/Gmail/Integrations/Gmail/Gmail.yml#L27) for reference